src/styles/input.css

```
@@ -3817,11 +3817,50 @@
   white-space: nowrap;
+  align-self: center;
+}
+
+.cs-sats-symbol {
+  display: inline-block;
+  position: relative;
+  flex: 0 0 auto;
+  width: 0.72em;
+  height: 1em;
+  margin-left: 0.24em;
+  color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)